mmtk/src/api.rs

```diff
@@ -4,23 +4,31 @@ use std::ffi::CStr;
 use mmtk::memory_manager;
 use mmtk::util::{Address, OpaquePointer, ObjectReference};
 use mmtk::Allocator;
-use mmtk::{SelectedTraceLocal, SelectedCollector, SelectedPlan};
+use mmtk::SelectedPlan;
 use mmtk::Mutator;
 use mmtk::Plan;
+use mmtk::MMTK;
+use mmtk::scheduler::*;
 use JikesRVM;
 use JTOC_BASE;
 use SINGLETON;
 use collection::BOOT_THREAD;
 use collection::VMCollection;
 
+#[no_mangle]
+pub extern "C" fn release_buffer(ptr: *mut Address) {
+    let _vec = unsafe { Vec::<Address>::from_raw_parts(ptr, 0, 4096) };
```

Comment:
By design, the value `4096` is the default buffer size and can be changed to other values. But currently, for jikesrvm, this is fixed to 4096 to match the hard-coded value in Java code. I've put the value in a global constant and use this constant instead of integer literals in rust code.